### PR TITLE
Update what-info-can-your-company-see-when-you-enroll-your-device-in-…

### PR DESCRIPTION
**Pull request by msbemba**
…intune.md

Added more information in app inventory details such as " Your organization can't see any data, such as texts, emails, and pictures, in your personal apps. The MTD app does report information about your apps, such as name and version, to your organization. Your organization can view all details about Managed apps."

Reference: - https://learn.microsoft.com/en-us/mem/intune/user-help/set-up-mobile-threat-defense#information-your-organization-can-see

Fixes #https://github.com/MicrosoftDocs/memdocs/issues/3317

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: memdocs/intune/user-help/what-info-can-your-company-see-when-you-enroll-your-device-in-intune.md [hunk 1]
```diff
@@ -103,7 +103,7 @@ Your organization can put a missing, corporate-owned iPhone or iPad into *manage
 Your organization can require you to install a mobile threat defense (MTD) app. If you're required to install an MTD app on your device:   
 
 * Corporate-owned device: Your organization can view details about all apps on the device. 
-* Personal-owned device: Your organization can only view the details of your managed apps.  
+* Personal-owned device:  Your organization can't see any data, such as texts, emails, and pictures, in your personal apps. The MTD app does report information about your apps, such as name and version, to your organization. Your organization can view all details about Managed apps.
```
Comment (JohanFreelancer9) [inline memdocs/intune/user-help/what-info-can-your-company-see-when-you-enroll-your-device-in-intune.md:106]:
```suggestion
* Personal-owned device: Your organization can't see any data, such as texts, emails, and pictures, in your personal apps. The MTD app does report information about your apps, such as name and version, to your organization. Your organization can view all the details about managed apps.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@lenewsad Please review and sign off this PR. Thanks

**Review by lenewsad — APPROVED** [review]

**Comment by lenewsad** [comment]
#sign-off

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Pull request merged**